bin/scale_calculate.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+MAX_INSTANCES=9
+MIN_INSTANCES=5
+SCALE_STEP=2
+
+CPU_BUSY_THRESHOLD=300
+CPU_IDLE_THRESHOLD=220
+
+app_status=$(cf app catalog-web)
```

Comment:
This is for scaling harvest admin, right? Do we want catalog-web hardcoded here?

Reply:
Yes, this script is called from the Restart job's `check-and-renew` script but we need to change the app name here. Good catch.